distribution/scripts/jenkins-essentials.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -euo pipefail
+
+passwordFileLocation="$JENKINS_HOME/secrets/initialAdminPassword"
+passwordFileLocationDirectory="$( dirname "$passwordFileLocation" )"
+
+echo -n "Creating $passwordFileLocationDirectory... "
+mkdir -p "$passwordFileLocationDirectory"
+echo "Done."
+
+echo -n "Generating admin password... "
+echo $RANDOM | md5sum | cut -d ' ' -f 1 > "$passwordFileLocation"
+echo "Done. Password value stored in $passwordFileLocation file."
+
+JENKINS_ADMIN_PASSWORD="$( cat "$passwordFileLocation" )"
+
+# Intended for ease of development. By default, password is obviously *not* put in logs.
+if [[ -n $INSECURE_SHOW_ADMIN_PASSWORD ]]; then
```

Comment:
Not `= yes` ? Double-quotes around that?

Reply:
I do not *really* care about the value, only checking if that env car is defined actually.

Reply:
Just a bit of technical debt for the next person to try to pass `=no` and be confused when this is treated like `=yes`.